index.html

```diff
@@ -570,9 +575,23 @@ <h1>{{ chosen_domain == 'US' ? 'USA' :
       return _.defaults({value: (cur - prev) / deltaInterval}, d);
     })]);
   }
+  else if (isGrowthChart) {
+    // In this mode, We plot an x,y graph with x as the total count and 'y' as
+    // the weekly count (not the average for one day, unlike with 'deltas')
+    extract.series = _.map(extract.series, p => [p[0],_.map(p[1], (d, i) => {
+      var prevIndex = i - 7; // Use last week for 'x'
+      var cur = (d ? d.value : 0),
+          prev = (p[1][prevIndex] ? p[1][prevIndex].value : 0);
+      var point = _.defaults({ x: cur, y: (cur - prev) }, d);
+      delete point["value"]; // delete unwanted 'value' prop as we have x,y
+      return point;
+    })]);
+  }
   if (scale.startsWith('log')) {
+    // When using log scale, we need to make sure we don't end up with entries
+    // that are missing their 'value' (or 'x/y') properties, as that breaks it
     extract.series = _.map(extract.series, p => [p[0], _.map(p[1], d => {
-      return d && d.value && d.value > 0 ? d : null;
+      return d && (d.value || (d.x && d.y)) ? d : null;
```

Comment:
Note: I removed the `d.value > 0` check which didn't seem necessary.No

Reply:
The d.value > 0 was to avoid negative numbers that sometimes showed up in daily deltas, messing up log plots.  I'll go test it now.

Reply:
Yeah - missing the value >0 check causes a crash on this URL:

http://localhost:8000/#/?start=2%2F29%2F20&series=deaths&stat=daily&include=NY&top=0

Let's add the d.value > 0 check back (and to match you might want to verify x and y > 0 also even though the data source probably has no negatives right now).

Reply:
Good catch, I brought it back and added it for x & y